core/src/main/java/com/linecorp/armeria/common/AggregatedHttpResponse.java

```diff
@@ -198,12 +233,18 @@ static AggregatedHttpResponse of(ResponseHeaders headers, HttpData content, Http
      * @param headers the HTTP headers
      * @param content the content of the HTTP response
      * @param trailers the HTTP trailers
+     *
+     * @throws IllegalArgumentException if the status of the specified {@link ResponseHeaders} is
+     *                                  {@link HttpStatus#isInformational()}.
      */
     static AggregatedHttpResponse of(Iterable<ResponseHeaders> informationals, ResponseHeaders headers,
                                      HttpData content, HttpHeaders trailers) {
 
         requireNonNull(informationals, "informationals");
         requireNonNull(headers, "headers");
+        final HttpStatus status = headers.status();
+        checkArgument(!status.isInformational(), "status: %s (expected: a non-1xx status", status);
```

Comment:
Missing closing parenthesis:
```suggestion
        checkArgument(!status.isInformational(), "status: %s (expected: a non-1xx status)", status);
```